OpenSim/Simulation/Model/CoordinateSet.cpp

```diff
@@ -67,7 +38,6 @@ CoordinateSet::CoordinateSet(const CoordinateSet& aCoordinateSet):
  */
 void CoordinateSet::populate(Model& model)
 {
-    setModel(model);
     // Aggregate Coordinates owned by the Joint's into a single CoordinateSet
```

Comment:
What invokese setModel now, or is it not needed anymore?

Reply:
`setModel()` is not part of the `ModelComponent` interface.